model/Human-GEM.yml

```
@@ -21900,6 +21900,14 @@
       - charge: -2
       - inchis: ""
       - metFrom: "HMRdatabase"
+    - !!omap
+      - id: "MAM01639n"
+      - name: "dAMP"
+      - compartment: "n"
+      - formula: "C10H12N5O6P"
+      - charge: -2
+      - inchis: ""
+      - metFrom: "HMRdatabase"
```

Since this compartmentalized metabolite is being added now, and presuming it hasn't been previously removed from HMR2, I would suggest to remove the reference to that model in `metFrom`:
```suggestion
      - metFrom: ""
```

---

Thanks for the corrections. I agree.

---

Well, it looks like too much of a hassle now to remove this attribute. In the future, I'm hoping the source model field `metFrom` will be removed, so it's fine to leave things as they are.